src/past/translation/__init__.py

```diff
@@ -32,7 +32,10 @@
 Inspired by and based on ``uprefix`` by Vinay M. Sajip.
 """
 
-import imp
+try:
+    import imp
+except ImportError:
+    import importlib
```

Comment:
When both modules are available, it would be better to prefer `importlib` to avoid logging deprecation warnings from `imp`.

Reply:
@aij Have a look at https://github.com/PythonCharmers/python-future/pull/632 .